<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>散斑图像处理软件</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>461</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="dirLineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>251</width>
      <height>19</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="interactive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>选择参考图像</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>选择图像文件夹</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>参考图像显示：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>90</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileLineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>251</width>
      <height>19</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dirLineEdit_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>251</width>
      <height>19</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>选择输出文件夹</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>518</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuselect">
    <property name="title">
     <string>算法选择</string>
    </property>
    <widget class="QMenu" name="menuGLCM">
     <property name="title">
      <string>GLCM</string>
     </property>
     <addaction name="actionCorrelation"/>
     <addaction name="actionHomogeneity"/>
    </widget>
    <addaction name="actionNIPC"/>
    <addaction name="actionZNCC"/>
    <addaction name="separator"/>
    <addaction name="menuGLCM"/>
    <addaction name="separator"/>
    <addaction name="actionMSV"/>
   </widget>
   <widget class="QMenu" name="menupre">
    <property name="title">
     <string>预处理</string>
    </property>
    <addaction name="actionROI"/>
   </widget>
   <addaction name="menupre"/>
   <addaction name="menuselect"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNIPC">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>NIPC</string>
   </property>
  </action>
  <action name="actionZNCC">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ZNCC</string>
   </property>
  </action>
  <action name="actionCorrelation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>correlation</string>
   </property>
  </action>
  <action name="actionHomogeneity">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>homogeneity</string>
   </property>
  </action>
  <action name="actionMSV">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>MSV</string>
   </property>
  </action>
  <action name="actionROI">
   <property name="text">
    <string>ROI</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
